test/SourceKit/Indexing/index_effective_access_level.swift.response

```diff
@@ -0,0 +1,260 @@
+          key.usr: "s:28index_effective_access_level9PrivateEn33_2295DDF1454D6A6D9229E8222CD85214LLO1ayA2DmF",
+          key.line: 15,
+          key.column: 10,
+          key.effective_access_level: source.decl.attribute.fileprivate
+        },
+        {
+          key.kind: source.lang.swift.decl.enumelement,
+          key.name: "b",
+          key.usr: "s:28index_effective_access_level9PrivateEn33_2295DDF1454D6A6D9229E8222CD85214LLO1byA2DmF",
+          key.line: 16,
+          key.column: 10,
+          key.effective_access_level: source.decl.attribute.fileprivate
+        }
+      ],
+      key.attributes: [
+        {
+          key.attribute: source.decl.attribute.private
+        }
+      ],
+      key.effective_access_level: source.decl.attribute.fileprivate
+    },
+    {
+      key.kind: source.lang.swift.decl.extension.enum,
+      key.name: "PublicEn",
+      key.usr: "s:e:s:28index_effective_access_level8PublicEnO5puFooyyF",
+      key.line: 19,
+      key.column: 18,
+      key.entities: [
+        {
+          key.kind: source.lang.swift.ref.enum,
+          key.name: "PublicEn",
+          key.usr: "s:28index_effective_access_level8PublicEnO",
+          key.line: 19,
+          key.column: 18,
+          key.effective_access_level: source.decl.attribute.public
+        },
+        {
+          key.kind: source.lang.swift.decl.function.method.instance,
+          key.name: "puFoo()",
+          key.usr: "s:28index_effective_access_level8PublicEnO5puFooyyF",
+          key.line: 20,
+          key.column: 10,
+          key.effective_access_level: source.decl.attribute.public
+        }
+      ],
+      key.attributes: [
+        {
+          key.attribute: source.decl.attribute.public
+        }
+      ],
+      key.effective_access_level: source.decl.attribute.public
+    },
+    {
+      key.kind: source.lang.swift.decl.extension.enum,
+      key.name: "PublicEn",
+      key.usr: "s:e:s:28index_effective_access_level8PublicEnO5puFooyXeXeF",
+      key.line: 23,
+      key.column: 11,
+      key.entities: [
+        {
+          key.kind: source.lang.swift.ref.enum,
+          key.name: "PublicEn",
+          key.usr: "s:28index_effective_access_level8PublicEnO",
+          key.line: 23,
+          key.column: 11,
+          key.effective_access_level: source.decl.attribute.public
+        },
+        {
+          key.kind: source.lang.swift.decl.function.method.instance,
+          key.name: "puFoo()",
+          key.usr: "s:28index_effective_access_level8PublicEnO5puFooyXeXeF",
+          key.line: 24,
+          key.column: 17,
+          key.attributes: [
+            {
+              key.attribute: source.decl.attribute.public
+            }
+          ],
+          key.effective_access_level: source.decl.attribute.public
+        }
+      ],
+      key.attributes: [
+        {
+          key.attribute: source.decl.attribute.public
+        }
+      ],
+      key.effective_access_level: source.decl.attribute.public
```

Comment:
This may be how it's stored internally, but it is inaccurate: extensions should have no effective visibility of their own because they don't "exist" independently of what they extend. Where a shorthand access level is used, it is only giving the default access of its members. We debated this at length back in the day and that is the model that the core team wanted exposed to users.